src/main/java/domain/LottoResult.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoResult {
+    NO_WIN(0, -1, false),
+    FIRST(2000000000, 6, false),
+    SECOND(30000000, 5, false),
+    THIRD(1500000, 5, false),
+    FOURTH(50000, 4, true),
+    FIFTH(5000, 3, false);
+
+    private int prize;
```

Comment:
```suggestion
    private final int prize;
```

final 키워드를 붙여준다면 어떠한 장점이 있을까요?

Reply:
오 값의 재할당을 막을 수 있겠네요! 지적 감사합니다!

Reply:
오 값의 재할당을 막을 수 있겠네요! 감사합니다:)